File: fix_sidebar.js
const fs = require('fs');
let content = fs.readFileSync('src/components/Sidebar.tsx', 'utf-8');

// The incorrect sed replacements:
content = content.replace(/import { Moon,\s+/g, 'import { ');

// Wait, I actually NEED Moon imported from lucide-react.
// The lucide-react import starts at line 3 and goes to 35. 
// Let's just make sure it's in lucide-react. 

fs.writeFileSync('src/components/Sidebar.tsx', content);

File: fix_sidebar.py
import os
import re

file_path = 'src/components/Sidebar.tsx'
with open(file_path, 'r') as f:
    code = f.read()

# Remove {(role === 'ADMIN' || role === 'SUPER_ADMIN') && (
code = code.replace("{(role === 'ADMIN' || role === 'SUPER_ADMIN') && (", "{true && (")

with open(file_path, 'w') as f:
    f.write(code)

print("Sidebar restrictions removed")
